src/main/app/claims/responseModelConverter.ts

```diff
@@ -170,6 +174,7 @@ export class ResponseModelConverter {
       defendant: this.convertPartyDetails(draft.defendantDetails),
       paymentIntention: draft.partialAdmission.paymentIntention && this.convertPaymentIntention(draft.partialAdmission.paymentIntention),
       freeMediation: FreeMediationUtil.convertFreeMediation(draft.freeMediation),
+      mediationPhoneNumber: mediationDraft.canWeUse.mediationPhoneNumber,
```

Comment:
in this `canWeUse` can be null if feature is not enabled, please handle that gracefully.

Reply:
It's handled now. As i said earlier, i have created new branch now and new PR and cherry picked commits and missed some changes...by the time i corrected it you are too quick to review :)